students/lauraannf/Lesson03/strformat_lab.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep 18 17:00:52 2018
+
+@author: Laura.Fiorentino
+"""
+print('----------Task One--------')
+testtuple = (2, 123.4567, 10000, 12345.67)
```

Comment:
Separate words with underscores per PEP8: test_tuple = for this as well as below